spec/lib/importers/licence_transaction/licence_importer_spec.rb

```diff
@@ -0,0 +1,185 @@
+require "spec_helper"
+require "gds_api/test_helpers/publishing_api"
+require "importers/licence_transaction/licence_importer"
+
+RSpec.describe Importers::LicenceTransaction::LicenceImporter do
+  let(:new_content_id) { "0cc89dd8-1055-4e6b-8f64-9a772dbe28db" }
+
+  before do
+    allow(SecureRandom).to receive(:uuid).and_return(new_content_id)
+
+    stub_publishing_api_has_content(
+      publishing_api_response,
```

Comment:
Minor, it's feels a little strange that `publishing_api_response` is defined in every test but used in the global stub. I initially looked for a method similar to `publishing_api_licences_response` and got a little confused when I couldn't find it.

It might be easier to read if a default value was set here, i.e. 
```
let(:publishing_api_response) { publishing_api_licences_response }
```

This value could be modified directly in the tests as needed.

Reply:
Sure - I've moved the `let` to have a default value at the global level 👍